install/kubernetes/helm/istio/charts/mixer/templates/statsdtoprom.yaml

```diff
@@ -50,7 +50,11 @@ spec:
     {{- end }}
       containers:
       - name: {{ $statsdname }}
-        image: "{{ $.Values.prometheusStatsdExporter.repository }}:{{ $.Values.prometheusStatsdExporter.tag}}"
+    {{- if eq $.Values.global.platform "amd64" }}
+        image: "{{ $.Values.prometheusStatsdExporter.repository_ppc64le }}:{{ $.Values.prometheusStatsdExporter.tag}}"
```

Comment:
How about use multi-arch docker image? Thus we can use one image name for different platforms.

Reply:
We want to use a multi-arch docker image but prom/statsd-exporter doesn't have multi-arch manifest support yet. That's why two different repositories for `amd64` and `ppc64le`.

Reply:
Can we write the manifest to do this?

Reply:
@gyliu513 the problem is we don't have any control over those registries. 2-3 months back we did suggest to add dockerfiles for `ppc64le` to prometheus, but its been pending since. That's the reason we didn't pursue the manifest route for these images.

Reply:
@hkshaw1990 I'm not sure what the process is for adding Dockerfiles for different arches.  Can you provide a link to the Dockerfile PR that has been pending?

@ldemailly @costinm would you mind weighing in on this specific point (do we want to add these Dockerfiles to the istio registries) so we can move this PR along?

Reply:
@hkshaw1990 can you please show me the issue that we opened in prometheus community?

Reply:
@gyliu513 yeah, this is a link to one of our PRs :
https://github.com/prometheus/prometheus/pull/2693

Reply:
If the istio project wanted to host manifest list that just points to those two, that would be doable. You'd just have a little extra overhead of keeping that list current, and I donno if it's worth it to save two lines of yaml.

Reply:
also -- is this if condition flipped? i think the action has the wrong arch on this one.

Reply:
@clnperez true, missed that. Will add a fix with the next rebase. Thanks!